config/routes.rb

```diff
@@ -1,15 +1,51 @@
 Rails.application.routes.draw do
+  devise_for :users, :controllers => { :registrations => "user/registrations" }
+  resources :users
   resources :member_meetings
   resources :calendarevents
   get"home", to: "pages#home"
   get"about", to: "pages#about"
   get"officer-team", to: "pages#officer-team"
   get"general-meetings", to: "pages#general-meetings"
-  get"admin", to: "pages#admin"
   get"outreach", to: "pages#outreach"
   get"social-events", to: "pages#social-events"
   get"career-development", to: "pages#career-development"
 
+  get"portal", to: "memberportal#member"
+  post"portal", to: "attendances#create"
+  get"member/calendar", to: "calendarevents#member"
+  get"member/meetings", to: "meetings#member"
+  get"member/points", to: "points#member"
+

```

Comment:
if we changed member to portal, should we get rid of the other paths off of member? I can look more into this

Reply:
This can be seen as a bug that we can account for in the turnover set up. I know what to do and how we should organize the code. Passes DoD and CoA